### PR TITLE
Access levels roles

### PR DESCRIPTION
**Pull request by nicoolle**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/userPage/ActiveMembership/ActiveMembership.tsx [hunk 1]
```diff
@@ -82,7 +82,10 @@ const ActiveMembership = () => {
       notificationLogic("error", error.message);
     });
 
-    setAccessLevels(await activeMembershipApi.getAccessLevelById(userId));
+    await activeMembershipApi.getAccessLevelById(userId).then(async (response) => {
+      console.log(response);
+      setAccessLevels(response)
```
Comment (OleksandrBilan) [inline src/pages/userPage/ActiveMembership/ActiveMembership.tsx:87]: Where is the semicolon?

File: src/pages/userPage/ActiveMembership/ActiveMembership.tsx [hunk 2]
```diff
@@ -82,7 +82,10 @@ const ActiveMembership = () => {
...
+    })
```
Comment (OleksandrBilan) [inline src/pages/userPage/ActiveMembership/ActiveMembership.tsx:88]: Same here

File: src/pages/userPage/ActiveMembership/ActiveMembership.tsx [hunk 3]
```diff
@@ -82,7 +82,10 @@ const ActiveMembership = () => {
       notificationLogic("error", error.message);
     });
 
-    setAccessLevels(await activeMembershipApi.getAccessLevelById(userId));
+    await activeMembershipApi.getAccessLevelById(userId).then(async (response) => {
```
Comment (OleksandrBilan) [inline src/pages/userPage/ActiveMembership/ActiveMembership.tsx:85]: You don't need the brackets around the 'response' variable

File: src/components/PrivateLayout/PrivateLayout.tsx [hunk 4]
```diff
@@ -167,7 +168,7 @@ const PrivateLayout = ({ children }: any) => {
                 </Menu.Item>
                 ) : (<> </>)
               }
-              {activeUserProfile?.city ? (
+              {(activeUserProfile || formerMember) ? (
```
Comment (OleksandrBilan) [inline src/components/PrivateLayout/PrivateLayout.tsx:171]: Wouldn't it be better to check if 'activeuserProfile' is not null or undefined?

File: src/pages/userPage/ActiveMembership/ActiveMembership.tsx [hunk 5]
```diff
@@ -82,7 +82,10 @@ const ActiveMembership = () => {
...
+      console.log(response);
```
Comment (tanyabiryuk) [inline src/pages/userPage/ActiveMembership/ActiveMembership.tsx:86]:
```suggestion
```

File: src/pages/userPage/ActiveMembership/ActiveMembership.tsx (hunk 3 above)
Comment (nicoolle) [reply src/pages/userPage/ActiveMembership/ActiveMembership.tsx:85]: In whole file before my changes is brackets. So maybe it`s better to leave like that?)

**Review by OleksandrBilan — APPROVED** [review]

**Review by christykobyletska — APPROVED** [review]

**Review by andrii52 — APPROVED** [review]

**Pull request closed without merging**